datasafe-simple-adapter/datasafe-simple-adapter-impl/src/main/java/de/adorsys/datasafe/simple/adapter/impl/SwitchableDatasafeServices.java

```diff
@@ -11,31 +12,32 @@
 import de.adorsys.datasafe.directory.api.config.DFSConfig;
 import de.adorsys.datasafe.encrypiton.api.types.encryption.EncryptionConfig;
 import de.adorsys.datasafe.simple.adapter.impl.cmsencryption.SwitchableCMSEncryptionModule;
-import de.adorsys.datasafe.simple.adapter.impl.pathencryption.LegacyCredentialsModule;
-import de.adorsys.datasafe.simple.adapter.impl.pathencryption.LegacyPathEncryptionModule;
+import de.adorsys.datasafe.simple.adapter.impl.pathencryption.SwitchablePathEncryptionModule;
 import de.adorsys.datasafe.simple.adapter.impl.profile.HardcodedProfileModule;
 import de.adorsys.datasafe.storage.api.StorageService;
 import de.adorsys.datasafe.types.api.context.overrides.OverridesRegistry;
 
 import javax.annotation.Nullable;
 import javax.inject.Singleton;
 
-/**
- * Datasafe service that is compatible with 0.4.3 release.
- */
 @Singleton
 @Component(modules = {
-        LegacyCredentialsModule.class,
+        DefaultCredentialsModule.class,
         DefaultKeyStoreModule.class,
         DefaultDocumentModule.class,
         SwitchableCMSEncryptionModule.class,
-        LegacyPathEncryptionModule.class,
+        SwitchablePathEncryptionModule.class,
         DefaultInboxActionsModule.class,
         DefaultPrivateActionsModule.class,
         HardcodedProfileModule.class,
         DefaultStorageModule.class
 })
-public interface LegacyDatasafeService extends DefaultDatasafeServices {
+public interface SwitchableDatasafeServices extends DefaultDatasafeServices {
+
+
+/**
+ * Datasafe service that is compatible with 0.4.3 release.
```

Comment:
Remove this comment, this service is not compatible with 0.4.3

Reply:
yes, forgotten